src/utilities/gql.ts

```diff
@@ -1405,6 +1406,14 @@ const gql = {
     }
   `,
 
+  SIMULATION_DATASET_ID: `#graphql
```

Comment:
Because this query is for getting the simulation dataset, can you rename this to `GET_SIMULATION_DATASET_ID` and move it up next to the other `GET_` queries, please?

Reply:
done